.vscodeignore

```diff
@@ -11,6 +11,7 @@ examples/**
 images/*.psd
 images/*.ai
 images/screenshot_folders.png
+images/screenshot890x486.gif
```

Comment:
I'm not sure either. They seem to be using a browser plugin, so I thought it was similar to an iframe, thus enabling the relative path. I'll check it.